website/source/docs/configuration/storage/consul.html.md

```diff
@@ -100,6 +100,13 @@ connection. You can read more about encrypting Consul connections on the
   [`ca_file`](https://www.consul.io/docs/agent/options.html#ca_file) setting in
   Consul.
 
+- `tls_ca_path` `(string: "")` – Specifies the path to a directory of
+  certificate authority files for Consul communication.  Is an alternative to
+  `tls_ca_file`.  This defaults to the system bundle if not specified.
```

Comment:
Duplicate tls_ca_file value.

Reply:
This is not a duplicate - i was trying to remark on the fact that`tls_ca_path` is used  when `tls_ca_file` is not supplied (see `rootcerts.LoadCACerts`). Any suggestions for improved wording?